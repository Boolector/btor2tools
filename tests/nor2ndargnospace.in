1 zero 32
2 nor 32 1 1nospace
