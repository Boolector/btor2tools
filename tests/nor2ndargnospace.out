*** catbtor: parse error in 'nor2ndargnospace.in' line 2: expected new-line (after two arguments)
